### PR TITLE
Add Shiba, rich markdown preview app

### PR DESCRIPTION
**Pull request by rhysd**
I suggest to add [Shiba](https://github.com/rhysd/Shiba) to apps in the page.

I created this app last year, brushed up this year, and released v1.0.0.  Some people already use this app and it looks to work fine on Linux, Windows and OS X.

### CODE REVIEW

**Comment by jlord** [comment]
Niiiice. Very into 🐶

**Comment by rhysd** [comment]
Thanks! :dog: 👍

File: _data/apps.yml [hunk 1]
```diff
@@ -1155,3 +1155,13 @@
...
   icon: "anatine.png"
+-
+  name: "Shiba"
+  description: "Rich markdown live preview app with linter"
+  repository: "https://github.com/rhysd/Shiba"
+  keywords:
+    - "markdown"
+    - "preview"
+    - "linter"
+  license: "MIT"
```
Comment (amitmerchant1990) [inline _data/apps.yml]: Add `website` parameter.

Comment (rhysd) [inline _data/apps.yml]: @amitmerchant1990 

CONTRIBUTING.md says

> Your website can be a repository with a README

Comment (rhysd) [inline _data/apps.yml]: #327 already offers the fix.

Comment (jlord) [inline _data/apps.yml]: Whoops, sorry I didn't catch that before, thanks!

**Pull request merged**